### PR TITLE
Update aptible to 0.8.5

### PR DESCRIPTION
**Pull request by krallin**
We updated this today: https://github.com/aptible/www.aptible.com/pull/459

Let me know if this is missing anything :)

Thanks!

----

*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

### CODE REVIEW

File: Casks/aptible.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 cask 'aptible' do
-  version '0.8.4+20170125122046'
-  sha256 '6cb9763d0ac9369241a3fc390762916cd46434572182dd268c70dc7b4c082719'
+  version '0.8.5+20170417215450'
+  sha256 'dc8d5f356ab3aaa9e532e63704021aadab2f2c82212aab57c13dfa922e3c3bd6'
 
   # omnibus-aptible-toolbelt.s3.amazonaws.com was verified as official when first introduced to the cask
-  url "https://omnibus-aptible-toolbelt.s3.amazonaws.com/aptible/omnibus-aptible-toolbelt/master/132/pkg/aptible-toolbelt-#{version.sub('+', '%2B')}-mac-os-x.10.11.6-1.pkg"
+  url "https://omnibus-aptible-toolbelt.s3.amazonaws.com/aptible/omnibus-aptible-toolbelt/master/134/pkg/aptible-toolbelt-#{version.sub('+', '%2B')}-mac-os-x.10.11.6-1.pkg"
```
Comment (numbermaniac) [inline Casks/aptible.rb]: If that `134` is a build number, that might need to become part of the `version` for simplicity. I'll leave it to one of the maintainers to decide how that should be done.

File: Casks/aptible.rb [hunk 2]
```diff
@@ -1,9 +1,9 @@
 cask 'aptible' do
-  version '0.8.4+20170125122046'
-  sha256 '6cb9763d0ac9369241a3fc390762916cd46434572182dd268c70dc7b4c082719'
+  version '0.8.5+20170417215450'
```
Comment (miccal) [inline Casks/aptible.rb]: I cannot see this version on the `homepage`?

File: Casks/aptible.rb (hunk 1 above)
Comment (krallin) [inline Casks/aptible.rb]: Thanks; this number is indeed a build number. I'm can adjust this as needed if there's a better approach to follow here.

File: Casks/aptible.rb [hunk 3]
```diff
@@ -1,13 +1,13 @@
 cask 'aptible' do
-  version '0.8.4+20170125122046'
-  sha256 '6cb9763d0ac9369241a3fc390762916cd46434572182dd268c70dc7b4c082719'
+  version '0.8.5+20170417215450,134'
+  sha256 'dc8d5f356ab3aaa9e532e63704021aadab2f2c82212aab57c13dfa922e3c3bd6'
 
   # omnibus-aptible-toolbelt.s3.amazonaws.com was verified as official when first introduced to the cask
-  url "https://omnibus-aptible-toolbelt.s3.amazonaws.com/aptible/omnibus-aptible-toolbelt/master/132/pkg/aptible-toolbelt-#{version.sub('+', '%2B')}-mac-os-x.10.11.6-1.pkg"
+  url "https://omnibus-aptible-toolbelt.s3.amazonaws.com/aptible/omnibus-aptible-toolbelt/master/#{version.after_comma}/pkg/aptible-toolbelt-#{version.sub('+', '%2B')}-mac-os-x.10.11.6-1.pkg"
```
Comment (krallin) [inline Casks/aptible.rb]: @vitorgalvao Thanks for your tweaks; I presume we'll need to use `version.before_comma.sub(...)` here?

**Comment by miccal** [comment]
Thank you for your contribution.

**Comment by krallin** [comment]
Thanks @miccal, @numbermaniac, @vitorgalvao ! 😄

**Pull request merged**